spec/ransack/search_spec.rb

```diff
@@ -93,9 +93,10 @@ module Ransack
         expect(condition.value).to eq 'Ernie'
       end
 
-      it 'preserves default scope conditions for associations' do
-        search = Search.new(Person, articles_title_eq: 'Test')
+      it 'preserves default scope and conditions for associations' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.